### DISCUSSION TITLE
Using MongoDB 4.2 [Synology]

### DISCUSSION THREAD

**Discussion by mietzen** [ViewTube/viewtube#2362] [General]
Hi,

I ran in to the problem of not having AVX support on my Synology NAS for running MongoDB 5, so I tried 4.2. 
It seems all to be running well, @moisout is there any Mongo DB 5 specifics that will cause any Issues when using 4.2?

**Comment by moisout** [comment]
There shouldn't be, i am not using any specific mongodb 5 features

**Reply by mietzen** [reply]
I opened a PR: https://github.com/ViewTube/wiki/pull/5

**Reply by moisout** [reply]
Thanks a lot!
